### ISSUE TITLE
Kubectl should offer context completions for config use-context

### ISSUE THREAD

**Issue by yissachar** [kubernetes/kubernetes#29386]
When trying to bash complete `kubectl config use-context` the completions offered are file/directory completions, which are not relevant to the command. Instead it should offer completions from the available kubectl contexts.

**Comment by surajssd** [comment]
I would like to work on it!

**Comment by superbrothers** [comment]
@yissachar Kubectl supported context completions for config use-context from v1.7.0-alpha.2. Can you close this issue?

**Comment by xiangpengzhao** [comment]
Will close it.

@thockin

**Comment by janetkuo** [comment]
This is broken by #44251:

```console
$ kubectl config use-context __kubectl_parse_config: command not found
```

@pwittrock fix this in 1.7? The fix is trivial (one line change)

**Comment by pwittrock** [comment]
@janetkuo If we can get a PR approved and lgtm'ed by Tuesday, lets try to get it in.

**Comment by superbrothers** [comment]
@pwittrock I fixes at #47293. Please see it.

**Comment by superbrothers** [comment]
@janetkuo Thank you for finding the issue.
